core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterDMTest.java

```diff
@@ -8296,50 +8321,6 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
     assertThat(toSql(root, DatabaseProduct.BIG_QUERY.getDialect()), isLinux(expectedSparkQuery));
   }
 
-  @Test public void testQualifyWithReferenceFromSelect() {
```

Comment:
Could you explain the reason for removing these test cases?

Reply:
Resolved